### ISSUE TITLE
[BUG] - `Select Wrapper` has AXE-Core violation in filter mode usinf description prop

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4007]
### JS Framework

all

### Porsche Design System version

3.29.0

### On which device(s) you can see the bug?

none

### Browser version

none

### Reproduction Example

Storefront

### Steps to reproduce the behavior

1. Go to https://designsystem.porsche.com/v3/components/select-wrapper/examples/#with-filter
2. Add `description` prop in DOM
3. Run AXE DevTools Browser Plugin

### Expected behavior

AXE Core should not fire a violation

### Relevant log output

```shell

```
